modules/cluster-and-ha-management/pages/ha-overview.adoc

```diff
@@ -22,12 +22,18 @@ TigerGraph HA provides continuous operation of some but not all services.
 Please note the following exceptions and consider your options for taking additional steps to maintain continuous operation or to restore service as quickly as possible.
 
 .If an HA system is operating with a failed node, unless the node is recovered or the system is reconfigured to xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[exclude that node], the following services are limited or unavailable:
-* A file-based data load cannot be loaded for the failed node.
-* A data partition slated for a kafka-based data load cannot be loaded.
-* New queries cannot be installed.
-* Schema changes are not allowed.
-* Backup and export operations are not available and will be rejected.
-* If the primary node is offline, access to Graph Studio is interrupted, but resumes once the primary node is back online.
+
+* A data partition slated for a connector-based loading, such as, s3 files or via kafka, *cannot* be loaded.
+
+* New queries *cannot* be installed.
++
+NOTE: However, new interpreted and any existing queries can still be executed.
+
+* Schema changes are *not* allowed.
+
+* Backup and export operations are *not available* and will be rejected.
+
+* If the primary node is offline, access to Graph Studio is *interrupted*, but resumes once the primary node is back online.
```

Comment:
Once 3.10 is released, we can remove this item.

Reply:
Agreed. And thank you. I created a new doc ticket to track improvement efforts for the HA overview. that ticket I linked it to the one associated with this PR but here is the link as well. https://graphsql.atlassian.net/browse/DOC-1974